bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonNetworkHandler.java

```diff
@@ -205,6 +206,20 @@ public void addMissingDevices(List<String> missing) {
         });
     }
 
+    public void deviceNotLinked(InsteonAddress addr) {
+        getThing().getThings().stream().forEach((thing) -> {
+            InsteonDeviceHandler handler = (InsteonDeviceHandler) thing.getHandler();
+            if (handler == null) {
+                throw new IllegalArgumentException("handler is null");
```

Comment:
This is not a correct exception to throw here. If handler is `null`, thing is not initialized yet. In that case you shouldn't interrupt the program flow by throwing an exception that will not be handled. Probably you should store this temporarily so you can tell the thing when it has been initialized? See for example #12113.

Reply:
This really can't happen just need to resolve the warning that is thrown.

Reply:
Why do you think it cannot happen? As far as I can see this method is called indirectly from the bridge handler itself (through `InsteonBinding`). If all child handlers are not initialized at this point, it will happen.

Reply:
Because the way the binding is written above, it waits at least 5 seconds after the last device has been created before the code that falls through this is even hit.

Reply:
In that case I guess you can simply skip those (instead of throwing), and perhaps add it comment why this is safe and cannot happen?

Reply:
done